From 8f4001dff968c593476f68559827202957ff2ff8 Mon Sep 17 00:00:00 2001
From: Nirmal Abraham <nabrah@codeaurora.org>
Date: Thu, 16 Nov 2017 16:04:52 +0530
Subject: [PATCH 01/16] sepolicy: rules to allow camera daemon access to app
 buffer

Add rule to allow mm-qcamera daemon to import the fd which
is allocated in app's context.

Change-Id: Icdc13cf34cef98a2529b79ee61900b5130585b0d

sepolicy: Allow camera daemon to access priv_app buffer.

Add rule to allow mm-qcamera daemon to import the fd which
is allocated in app's context. This is required for VT call
camera preview to work.

Change-Id: Iea4d82a44f42715ca888960549494e788dd99563
CRs-Fixed: 2133945

[mikeioannina]: Move to common sepolicy

Change-Id: I6e1c48df94b31132f5b1f9afa3a07ccc3c4aab4d
---
 common/mm-qcamerad.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/mm-qcamerad.te b/common/mm-qcamerad.te
index 687e809..5213494 100644
--- a/common/mm-qcamerad.te
+++ b/common/mm-qcamerad.te
@@ -40,7 +40,7 @@ allow mm-qcamerad self:process execmem;
 allow mm-qcamerad video_device:dir r_dir_perms;
 allow mm-qcamerad { gpu_device video_device sensors_device }:chr_file rw_file_perms;
 
-allow mm-qcamerad { surfaceflinger mediaserver cameraserver hal_camera }:fd use;
+allow mm-qcamerad { surfaceflinger mediaserver cameraserver hal_camera untrusted_app_all priv_app }:fd use;
 
 allow mm-qcamerad camera_data_file:dir w_dir_perms;
 #allow mm-qcamerad camera_data_file:sock_file { create unlink };
-- 
2.15.1

